api-admin/src/main/java/bio/terra/pearl/api/admin/controller/portal/PortalController.java

```diff
@@ -49,4 +50,11 @@ public ResponseEntity<List<PortalShallowDto>> getAll() {
             .collect(Collectors.toList());
     return ResponseEntity.ok(portalDtos);
   }
+
+  @Override
+  public ResponseEntity<Void> removePortalUser(String portalShortcode, UUID adminUserId) {
+    AdminUser operator = requestService.requireAdminUser(request);
+    portalExtService.removeUserFromPortal(adminUserId, portalShortcode, operator);
+    return ResponseEntity.status(204).build();
```

Comment:
```suggestion
    return ResponseEntity.noContent().build();
```

nit: for consistency and readability, we've been returning statuses using the status code name rather than the number